packages/api/src/controllers/access-control.ts

```diff
@@ -128,17 +125,18 @@ app.post(
 
     const playbackPolicyType = content.playbackPolicy?.type ?? "public";
 
-    if (user.createdAt < HACKER_DISABLE_CUTOFF_DATE) {
-      let limitReached = await freeTierLimitReached(content, user, req);
-      if (limitReached) {
-        throw new ForbiddenError("Free tier user reached viewership limit");
-      }
+    let response = {};
+
+    if (isFreeTierUser(user) && user.createdAt > HACKER_DISABLE_CUTOFF_DATE) {
+      response = {
+        rateLimit: MAX_ALLOWED_VIEWERS_FOR_FREE_TIER,
```

Comment:
A couple things regarding this schema:
- Since this schema is defined by Catalyst and implemented by the API, we should probably stick to Catalyst's design choices. Specifically, this should be using `snake_case` instead of` camelCase`
- "Rate limit" is a completely generic term and I think we should be more specific here. Specifically, I was thinking of having a `rate_limit` _object_ instead, and inside the object we define a specific limit we want _by name_.

e.g. this whole payload could be
```suggestion
        rate_limit: {
          max_view_starts_per_minute: MAX_ALLOWED_VIEWERS_FOR_FREE_TIER, 
        }  
```
(or sth like that, if the limit is actually something else)